xray/core/dataset.py

```diff
@@ -1116,6 +1118,56 @@ def relevant_keys(mapping):
                         zip(*[v for k, v in indexers])]],
                       dim=dim, coords=coords, data_vars=data_vars)
 
+    def sel_points(self, dim='points', method=None, **indexers):
+        """Returns a new dataset with each array indexed pointwise by tick
+        labels along the specified dimension(s).
+
+        In contrast to `Dataset.isel_points`, indexers for this method should
+        use labels instead of integers.
+
+        In contrast to `Dataset.sel`, this method selects points along the
+        diagonal of multi-dimensional arrays, not the intersection.
+
+        Parameters
+        ----------
+        dim : str or DataArray or pandas.Index or other list-like object, optional
+            Name of the dimension to concatenate along. If dim is provided as a
+            string, it must be a new dimension name, in which case it is added
+            along axis=0. If dim is provided as a DataArray or Index or
+            list-like object, its name, which must not be present in the
+            dataset, is used as the dimension to concatenate along and the
+            values are added as a coordinate.
+        method : {None, 'nearest', 'pad'/'ffill', 'backfill'/'bfill'}, optional
+            Method to use for inexact matches (requires pandas>=0.16):
+
+            * default: only exact matches
```

Comment:
I know these bullets are straight from `pandas` but "default" isn't a valid keyword [here](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.reindex.html#pandas.Series.reindex).  I think this may be clearer as `* None (default): only exact matches`.